stripe-core/src/main/java/com/stripe/android/core/utils/FeatureFlags.kt

```diff
@@ -10,6 +10,8 @@ object FeatureFlags {
     val nativeLinkAttestationEnabled = FeatureFlag("Native Link Attestation")
     val instantDebitsIncentives = FeatureFlag("Instant Bank Payments Incentives")
     val editSavedCardPaymentMethodEnabled = FeatureFlag("Edit Saved Card Payment Method")
+    val financialConnectionsLiteKillswitch = FeatureFlag("FC Lite killswitch")
```

Comment:
The feature flags default to `false`. Should this be `financialConnectionsLiteEnabled` instead?

Reply:
good point - that way we keep it disabled until releasing - https://github.com/stripe/stripe-android/pull/10504/commits/788dda569279d2997c45d54bd968871d5dbb0d1e